graphql-kickstart-spring-support/src/main/java/graphql/kickstart/spring/AbstractGraphQLController.java

```diff
@@ -34,9 +34,7 @@ public Object graphqlPOST(
       @Nullable @RequestBody(required = false) String body,
       ServerWebExchange serverWebExchange) throws IOException {
 
-    if (body == null) {
-      body = "";
-    }
+    body = body != null ? body : "";
```

Comment:
Instead of a `null` check you could also do `body = Optional.ofNullable(body).orElse("");` Although now that I look at this apparently we're re-assigning a parameter here, which isn't good practice to begin with. Besides that it's a pretty strange re-assignment too, since in my opinion it would probably be cleaner if the methods this value is passed to below are able to cope with a `null` value. That would eliminate the need for this re-assignment completely.